frontend/src/components/header/userMenu/styles.css

```diff
@@ -19,4 +19,5 @@
     color: black;
     text-align: center;
     margin-right: 1rem;
+    font-weight: 300;
```

Comment:
ok, i understand that you don't add parameter, so check every single css and add it